src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt

```diff
@@ -24,12 +24,50 @@ data class OsvSchema<D, A_D, A_E, A_R_D> (
 
     val id: String,
     @Serializable(with = LocalDateTimeRfc3339Serializer::class)
+    @JsonSerialize(
+        using = LocalDateTimeRfc3339JacksonSerializer::class,
+        contentUsing = JsonSerializerNone::class,
```

Comment:
according to https://youtrack.jetbrains.com/issue/KT-22704: the default values in annotations should work as default values in actual type -- but it doesn't